recipes/cwinpy/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "cwinpy" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 3caedf3aac6b8ef9bc0df8467e1610587c1fae3b2a0266ee6c839d45a79e5cb0
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - setuptools
+  run:
+    - python >=3.6
+    - astropy
+    - numpy >=1.15
+    - bilby >=0.5.8
+    - bilby_pipe >=0.2.0
+    - scipy
+    - matplotlib
+    - lintegrate
+    - numba
+    - gwpy
+    - pycondor
+    - lal
+    - lalpulsar
```

Comment:
@mattpitkin, can you alphabetise these dependencies, which should make it easier to read.